dist/post_run/index.js

```diff
@@ -48580,7 +48580,9 @@ function installLint(versionConfig) {
             repl = /\.zip$/;
         }
         else {
-            extractedDir = yield tc.extractTar(archivePath, process.env.HOME);
+            // We want to always overwrite files if the local cache already has them
```

Comment:
Did you change it manually? If so please run `npm run build` to generate the dist

Reply:
This was generated with `npm run build`.

But looks like fixing this may need to be done in actions/toolkit to be able to detect the tar variant being used.

But the problem there is that will create a difference in behavior, depending on what platform it is being run on. Not sure if that is a major concern or not, but could lead to confusion why things would work in one place but not another.

Reply:
ok, thanks for the clarification